packages/legend-application-studio/src/stores/editor/EditorConfig.ts

```diff
@@ -37,6 +37,7 @@ export enum ACTIVITY_MODE {
   WORKFLOW_MANAGER = 'WORKFLOW_MANAGER',
   TEST_RUNNER = 'TEST_RUNNER',
   REGISTER_SERVICES = 'REGISTER_SERVICES',
+  END_TO_END_WORKFLOWS = 'END_TO_END_WORKFLOWS',
```

Comment:
should be a separate enum because it will be a separate grouping from the activity bars.